packages/backend/src/database/migrations/20210825125410_order_saved_fields.ts

```diff
@@ -1,11 +1,13 @@
 import { Knex } from 'knex';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I prefer to use comments, rather than importing a new type from common on the migration files